src/main/python/deep_qa/layers/highway.py

```diff
@@ -0,0 +1,10 @@
+from keras.layers import Highway as KerasHighway
+
+class Highway(KerasHighway):
+    """
+    Keras's Highway layer does not support masking, but it easily could, just by returning the
```

Comment:
`Keras's` -> `Keras'`? I don't quite mind either way. Backticks on `Highway`.
What do you think about naming this `MaskedHighway?`